themes/jb/layouts/partials/sponsor/medium.html

```diff
@@ -1,7 +1,8 @@
-<p style="max-width: 75ch;"><strong>{{- .Title }}</strong> - {{ .Params.description -}}
+<p style="max-width: 75ch;">
+    <strong>{{- .Title }}</strong> - {{ replaceRE (print "^" .Title "(?: is)? (.*)") "$1" .Params.description -}}
```

Comment:
For this I'd put a comment like below, so we come back and remove the regex once we have a more website mindful ads on the JB episodes.

```suggestion
    <!-- TODO: remove after ads are more mindful about website and no longer include the name of the sponsor as the first word of the description -->
    <strong>{{- .Title }}</strong> - {{ replaceRE (print "^" .Title "(?: is)? (.*)") "$1" .Params.description -}}
```